.circleci/config.yml

```diff
@@ -469,6 +469,39 @@ jobs:
           name: Test notebooks
           command: make test_notebooks
 
+  push_docs:
+    docker:
+    - image: cimg/python:3.8
+    steps:
+      - checkout
+      - run:
+          name: Update Submodules
+          command: git submodule update --init
+      - setup_environment_mpi
+      - run:
+          name: Install doc requirements
+          command: |
+            . venv/bin/activate
+            pip install -r requirements_docs.txt -c constraints.txt
+      - run:
+          name: Make docs
+          command: |
+            . venv/bin/activate
+            make docs
+      - run:
+          name: Move into place
+          command: |
+            git clone --branch gh-pages git@github.com:ai2cm/pace.git gh-pages
```

Comment:
Can you merge this plan with the doctest plan? These should only be pushed if doctests pass anyways.

Reply:
As far as I can tell these two plans are doing different things, and should be run at different times, so I'm not sure what commonality there is to merge them. The new plan `push_docs` builds and pushes docs when a PR is merged to `main`, while the `doctest` plan tests that the sphinx docs build and the snippets run without error.

Reply:
Right, this will only run if the `doctest` plan passes, since that is a branch protection requirement.